### DISCUSSION TITLE
Help... "fatal error" in cpeditor

### DISCUSSION THREAD

**Discussion by ghost** [cpeditor/cpeditor#1342] [Q&A]
I tried to compile my code and suddenly this appeared:

[
c++: fatal error: cannot execute 'ld': CreateProcess: No such file or directory
compilation terminated. 
]

that I have to do?

**Comment by ouuan** [comment]
It's a problem with your compiler configurations. Please try https://www.google.com/search?q=fatal+error%3A+cannot+execute+%27ld%27%3A+CreateProcess%3A+No+such+file+or+directory first.
